app/Services/TransactionImport/TransactionPersister.php

```
@@ -0,0 +1,170 @@
+use App\Contracts\Import\BatchResultInterface;
+use App\Models\Transaction;
+use App\Services\Csv\CsvProcessResult;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+
+/**
+ * Handles persisting transactions to the database.
+ */
+class TransactionPersister
+{
+    private array $batchQueue = [];
```

PHPStan (2.1.15)

Property App\Services\TransactionImport\TransactionPersister::$batchQueue type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)